src/screens/app/Components/UserCaption/UserCaption.js

```diff
@@ -1,23 +1,32 @@
 import React from 'react';
 import {Subheading} from 'react-native-paper';
 import {Text, StyleSheet} from 'react-native';
+import PropTypes from 'prop-types';
 
 export default class UserCaption extends React.Component {
   constructor(props) {
     super(props);
-  }
 
-  handleCommentClick = () => {};
+    this.state = {
+      caption: this.props.caption,
+      username: this.props.username,
+    };
+  }
```

Comment:
As noted in the test file, what is the reason behind storing state? When will we need it for data manipulation?

Reply:
Will be converting it to props! Thanks again for noticing